src/app/components/GameModeName.js

```diff
@@ -0,0 +1,21 @@
+import validateString from '../utils/validateString';
+
+export default function gameMode(message) {
+  const isAValidString = validateString(message);
+  if (!isAValidString) {
+    throw new Error('Not a valid string');
+  }
+
+  const modeInfo = document.createElement('div');
+  const modeTextElement = document.createElement('span');
+  const questionElement = document.createElement('span');
+
+  modeInfo.id = 'mode-info';
+  modeInfo.className = 'mode-info';
+  modeTextElement.className = 'mode-info__mode-text';
```

Comment:
maybe mode-info__text? Although since the cards are repeated we could consider more universal class names.

Reply:
Yeah, looks better